src/explore-education-statistics-admin/src/pages/release/components/PreReleaseUserAccessForm.tsx

```diff
@@ -86,7 +86,14 @@ const PreReleaseUserAccessForm = ({
           validationSchema={Yup.object<FormValues>({
             email: Yup.string()
               .required('Enter an email address')
-              .email('Enter a valid email address'),
+              .email('Enter a valid @education.gov.uk email address')
+              .test({
+                name: 'email format',
+                message: 'Enter a valid @education.gov.uk email address',
+                test: (value: string) =>
+                  value.split('@').length === 2 &&
```

Comment:
Relatively minor, but would just extract the parts into a new variable first so that we're not doing the split twice.